libraries/ms-helper/inc/exported_enums.h

```diff
@@ -24,3 +24,28 @@ typedef enum {
   EE_CHARGER_CONN_STATE_CONNECTED,
   NUM_EE_CHARGER_CONN_STATES,
 } EEChargerConnState;
+
+typedef enum {
```

Comment:
@ckitagawa - I can see this becoming very messy very quickly. We should probably tag these with the messages they're intended to be used with or something.

Reply:
I intended to do this integration after the race. I understand it might get messy but I want to keep development velocity relatively high and re-rolling/updating codegen-tooling as frequently as this would require would get troublesome. I also have other things I'd rather prioritize than this. I'm willing to take the hit on technical debt for now unless you strongly disagree.

Reply:
SGTM. I've started adding a comment before my exported enums with the message they're intended to be used with. I think that helps.